src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,397 @@
+				e.setIsTerminated(false);
+				break;
+			}
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return e;
+	}
+
+	/**
+	 * <p>Calls different user methods depending upon user choice</p>
+	 * 
+	 * @param normalUserChoice
+	 * @param userName
+	 * @author srijan
+	 */
+	public void normalUserOptionHandler(int normalUserChoice, Employee em,Scanner input) {
+
+		switch (normalUserChoice) {
+		case 1:
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		case 2:
+			employee = employeeControl.findUserByUserName(em);
+			LOGGER.log(Level.INFO, "{0}", employee);
+			Employee changedProfileDetails = changeProfile(employee,input);
+			Boolean status =
+				employeeControl.updateProfile(changedProfileDetails);
+			if (status.equals(true)) {
+				LOGGER.log(Level.INFO, "Update Successful");
+			}
+			break;
+		case 3:
+			exit();
+			break;
+		default:
+			LOGGER.log(Level.INFO,
+				"Please choose option properly,Default view all user block");
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		}
+	}
+
+	/**
+	 * <p>edit and terminate user</p>
+	 * 
+	 * @return
+	 * @author srijan
+	 */
+	public int editAndTerminateEmployee(Scanner input) {
+
+		int employeeIdToEdit = 0;
+		try {
+			input = new Scanner(System.in);
+			employeeIdToEdit = input.nextInt();
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return employeeIdToEdit;
+	}
+
+	/**
+	 * <p>Change profile of an user</p>
+	 * 
+	 * @param emp
+	 * @return
+	 * @author srijan
+	 */
+	public Employee changeProfile(Employee emp,Scanner input) {
+
+		try {
+			input = new Scanner(System.in);
+			LOGGER.log(Level.INFO, "Enter Username:");
+			this.username = input.nextLine();
+			if (!"".equals(username))
+				emp.setUserName(username);
+			LOGGER.log(Level.INFO, "Enter Fullname:");
+			this.fullname = input.nextLine();
+			if (!"".equals(fullname))
+				emp.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department:");
+			this.department = input.nextLine();
+			if (!"".equals(department))
+				emp.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			this.address = input.nextLine();
+			if (!"".equals(address))
+				emp.setAddress(address);
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		return emp;
+	}
+
+	/**
+	 * <p>Takes input from user the search choice and value of choice</p>
+	 * 
+	 * @return
+	 */
+	public SearchEmployee choiceOfSearch(Scanner input) {
+
+		SearchEmployee emp = new SearchEmployee();
+		int choice = 0;
+		try {
+			input = new Scanner(System.in);
+			choice = input.nextInt();
+			if (choice == 1)
+				emp.setSearchChoice("department");
+			else if (choice == 2)
+				emp.setSearchChoice("fullname");
+			else if (choice == 3)
+				emp.setSearchChoice("address");
+			else
+				emp.setSearchChoice("default");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "default" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)